public/app/plugins/datasource/azuremonitor/azure_log_analytics/azure_log_analytics_datasource.ts

```diff
@@ -174,6 +175,21 @@ export default class AzureLogAnalyticsDatasource extends DataSourceWithBackend<
     return target;
   }
 
+  private expandResourcesForMultipleVariables(resources: string[] | undefined, scopedVars: ScopedVars, templateSrv: TemplateSrv): string[] | undefined {
```

Comment:
What's the thinking for making this private? I wonder if we make this a util function and update other areas of the codebase to make sure this works elsewhere, wdyt?